jkube-kit/enricher/specific/src/test/java/org/eclipse/jkube/kit/enricher/specific/WebAppHealthCheckEnricherTest.java

```diff
@@ -21,64 +21,68 @@
 
 import org.eclipse.jkube.kit.enricher.api.JKubeEnricherContext;
 import org.eclipse.jkube.kit.enricher.api.model.Configuration;
-import mockit.Expectations;
-import mockit.Mocked;
+import org.junit.Before;
 import org.junit.Test;
+import org.mockito.MockedConstruction;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockConstruction;
+import static org.mockito.Mockito.when;
 
 public class WebAppHealthCheckEnricherTest {
 
-    @Mocked
     private JKubeEnricherContext context;
-
+    @Before
+    public void setUp() {
+        context = mock(JKubeEnricherContext.class);
+    }
     private void setupExpectations(Map<String, Object> config) {
-        new Expectations() {{
-            context.hasPlugin("org.apache.maven.plugins", "maven-war-plugin");
-            result = true;
-
-            Configuration.ConfigurationBuilder configBuilder = Configuration.builder();
-            configBuilder.pluginConfigLookup(getProjectLookup(config));
-        }};
+        when(context.hasPlugin("org.apache.maven.plugins", "maven-war-plugin")).thenReturn(true);
+        Configuration.ConfigurationBuilder configBuilder = Configuration.builder();
+        configBuilder.pluginConfigLookup(getProjectLookup(config));
     }
 
     @Test
     public void noEnrichmentIfNoPath() {
 
         // given
 
-        WebAppHealthCheckEnricher enricher = new WebAppHealthCheckEnricher(context);
-        setupExpectations(new HashMap<>());
+        try (MockedConstruction<WebAppHealthCheckEnricher> helmUploaderMockedConstruction = mockConstruction(WebAppHealthCheckEnricher.class)){
+            WebAppHealthCheckEnricher enricher = new WebAppHealthCheckEnricher(context);
+            setupExpectations(new HashMap<>());
+            // when
 
-        // when
+            Probe probeLiveness = enricher.getLivenessProbe();
+            Probe probeReadiness = enricher.getReadinessProbe();
 
-        Probe probeLiveness = enricher.getLivenessProbe();
-        Probe probeReadiness = enricher.getReadinessProbe();
+            // then
+            assertThat(probeLiveness).isNull();
+            assertThat(probeReadiness).isNull();
+        }
 
-        // then
-        assertThat(probeLiveness).isNull();
-        assertThat(probeReadiness).isNull();
     }
 
     @Test
     public void enrichmentWithDefaultsIfPath() {
 
         // given
+        try (MockedConstruction<WebAppHealthCheckEnricher> helmUploaderMockedConstruction = mockConstruction(WebAppHealthCheckEnricher.class)) {
```

Comment:
I'm completely unsure what helm has to do in this test (seems like a copy-paste legacy)
Also, mocking the class at test is not good. Any specific reason for this?